packages/elements/src/Text/utils/styles/addMarginsToLists.ts

```diff
@@ -0,0 +1,29 @@
+const listMargins = (node: Element): void => {
+  const allowedTags = ["UL", "OL"];
+  if (allowedTags.includes(node.nodeName)) {
+    const { marginTop, marginBottom } = window.getComputedStyle(node);
+
+    if (marginTop) {
+      node.firstElementChild?.classList.add(`brz-mt-lg-${marginTop}`);
+    }
+    if (marginBottom) {
+      node.lastElementChild?.classList.add(`brz-mb-lg-${marginBottom}`);
+    }
+  } else if (node.nodeType === Node.ELEMENT_NODE) {
+    const children = node.childNodes;
```

Comment:
i think we need to parse `children` not `childNodes` childnodes contain the text, comments etc.. but we need only the Elements like ul, li